src/Adapter/Adapter.php

```diff
@@ -49,8 +49,6 @@ class Adapter implements AdapterInterface, Profiler\ProfilerAwareInterface
     /** @var ResultSet\ResultSetInterface */
     protected $queryResultSetPrototype;
 
-    /** @var Driver\StatementInterface */
-    protected $lastPreparedStatement;
```

Comment:
Overall OK with deprecating this, not with removing it, as it may be a minor BC break.

Fine with no longer assigning anything to it, since it may have been `null` at any point in time already